models/auth/webauthn_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package auth
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"code.gitea.io/gitea/models/unittest"
+
+	"github.com/duo-labs/webauthn/webauthn"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetWebAuthnCredentialByID(t *testing.T) {
+	assert.NoError(t, unittest.PrepareTestDatabase())
+
+	res, err := GetWebAuthnCredentialByID(1)
+	assert.NoError(t, err)
+	assert.Equal(t, "WebAuthn credential", res.Name)
+
+	_, err = GetWebAuthnCredentialByID(342432)
+	assert.Error(t, err)
+	assert.True(t, IsErrWebAuthnCredentialNotExist(err))
+}
+
+func TestGetWebAuthnCredentialsByUID(t *testing.T) {
+	assert.NoError(t, unittest.PrepareTestDatabase())
+
+	res, err := GetWebAuthnCredentialsByUID(1)
```

Comment:
should be 32 - see comment on the fixture.

```suggestion
	res, err := GetWebAuthnCredentialsByUID(32)
```